scripts/generate-integration-pages.ts

```diff
@@ -178,6 +179,41 @@ function absoluteLinks({ base }: { base: string }) {
 	};
 }
 
+/** Remark plugin to replace GitHub . */
```

Comment:
Nit: this comment is complete!

```suggestion
/** Remark plugin to replace GitHub note/warning syntax with docs-style asides. */
```